mungegithub/submit-queue/Makefile

```diff
@@ -0,0 +1,24 @@
+KUBECONFIG ?= $(HOME)/.kube/config
+
+# Updates nginx https proxy and the ingress with the local nginx-redirect/nginx.conf and ingress.yaml files.
+push-network:
+	# Replace the nginx config and restart the pod.
+	kubectl --kubeconfig=$(KUBECONFIG) delete cm nginx-https-redirect && \
```

Comment:
configmap and pod have better readability than cm and po. 

I recommend we use the full name for instructions and commands we check into the repo.